src/server/handlers/robots-handler.ts

```diff
@@ -16,5 +16,7 @@ export default async ({ res }: { res: Response }) => {
   Disallow: /password_change
   Disallow: /search/
   Disallow: /modlog
-  `);
+  
+  User-Agent: *
+  Crawl-delay: 60`);
```

Comment:
Couldn't this go right beneath the existing `User-Agent: *` from line 6?

Reply:
True, fixed